automation/queue/select_batch.py

```
@@ -0,0 +1,370 @@
+        --output-jsonl output/selected.jsonl \
+        --output-files output/selected_files.txt
+
+Notes:
+- The metadata entries should have at least a `file` field. Optional fields used for filtering:
+  `categories` (list[str] or str), `region` (str), `difficulty` (str), and `embedding` (list[float]).
+- Multiple --processed inputs are supported and can be .jsonl, .json, or .txt (newline-separated paths).
+- If GITHUB_OUTPUT env var is present, this script writes a few outputs: selected_count and files_csv.
+"""
+from __future__ import annotations
+
+import argparse
```

Ruff (0.12.2)

`argparse` imported but unused

Remove unused import: `argparse`

(F401)